app/lib/efile/az/az140.rb

```diff
@@ -234,13 +234,14 @@ def calculate_line_53
 
 
       def calculate_line_56
-        if !@direct_file_data.claimed_as_dependent? && @intake.qualified_for_excise_credit? && @intake.ask_whether_incarcerated?
+        if @intake.qualified_for_excise_credit?
```

Comment:
idea: 
- add method to intake called something like `disqualified_for_excise_credit_federal?`
- call `disqualified_for_excise_credit_federal?` in controller show
- add method to intake called something like `disqualified_for_excise_credit_state?`
- call both methods here?